openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -138,58 +138,53 @@
 - control_key: AU-5
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - key: a
-      text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-161
+      text: |      
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
+ 
     - key: b
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-161
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
 
 - control_key: AU-5 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-162
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
 
 - control_key: AU-5 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-163
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
 
 - control_key: AU-5 (3)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-164
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
 
 - control_key: AU-5 (4)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-165
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
```

Comment:
Similar to other controls, a response for AU-5(4) will need to document what fail-secure options could be if the cluster logging operator (+associated services) fail. Today that is not possible, hence the link to OpenShift product Jira.

NIST ref link:
https://nvd.nist.gov/800-53/Rev4/control/AU-5#enhancement-4

Language:

AU-5(4) | RESPONSE TO AUDIT PROCESSING FAILURES \| SHUTDOWN ON FAILUREThe information system invokes a [Selection: full system shutdown; partial system shutdown; degraded operational mode with limited mission/business functionality available] in the event of [Assignment: organization-defined audit failures], unless an alternate audit capability exists.
-- | --